appcues/src/main/java/com/appcues/ui/presentation/ViewPresenter.kt

```diff
@@ -73,6 +89,9 @@ internal abstract class ViewPresenter(
     private fun onCompositionDismiss() {
         AppcuesActivityMonitor.unsubscribe(activityMonitorListener)
 
+        gestureListener?.clearListener()
+        gestureListener = null
```

Comment:
This addition is so we don't keep listening to shake after a preview is dismissed. Previously, it would just listen indefinitely, giving haptic feedback, but not doing anything, which _i think_ is undesired.